services/gcn_service/gcn_service.py

```diff
@@ -1,23 +1,20 @@
-import yaml
-
 import gcn
+from sqlalchemy.orm import sessionmaker, scoped_session
 
 from baselayer.log import make_log
+from baselayer.app.models import init_db
+from baselayer.app.env import load_env
 
-from skyportal.tests import api
-
+from skyportal.handlers.api.gcn import post_gcnevent
+from skyportal.models import (
+    DBSession,
+)
 
-def get_token():
-    try:
-        token = yaml.load(open('.tokens.yaml'), Loader=yaml.Loader)['INITIAL_ADMIN']
-        print('Token loaded from `.tokens.yaml`')
-        return token
-    except (FileNotFoundError, TypeError, KeyError):
-        print('Error: no token specified, and no suitable token found in .tokens.yaml')
-        return None
+env, cfg = load_env()
 
+init_db(**cfg['database'])
 
-admin_token = get_token()
+Session = scoped_session(sessionmaker(bind=DBSession.session_factory.kw["bind"]))
```

Comment:
Could probably use DBSession directly, curious why this is necessary.

Reply:
this was just copying from gcn.py, I removed.